roboflow/util/image_utils.py

```diff
@@ -3,10 +3,14 @@
 import os
 import urllib
 
+import pillow_heif
 import requests
 import yaml
 from PIL import Image
 
+pillow_heif.register_heif_opener(thumbnails=False)  # Register for HEIF/HEIC
+pillow_heif.register_avif_opener()  # Register for AVIF
```

Comment:
I don't understand this thumbnails arg, but I think it should be false here too.

```suggestion
pillow_heif.register_avif_opener(thumbnails=False)  # Register for AVIF
```